### ISSUE TITLE
colour minimap 

### ISSUE THREAD

**Issue by Thermatix** [severin-lemaignan/vim-minimap#46]
Is there a way to get the minimap to mimic the current buffer's text colouring?

**Comment by naught101** [comment]
Yeah, this would make things much more readable. I guess it's not exactly possible, because each "row" in the mimi-map is actually just one quarter of a font-row, so the colours could only be changed every 4 rows.

But it might still be ok to just take the colour of the first row of a 4-row set, and apply that, or take the most common colour of the four rows and use that... The first option would be useful for python, for example, where the coding style guidlines suggest using 2 empty rows before a function/class, which would make the `def`/`class` colouring stand out in the minimap, and make it a fair bit easier to discover functions.

**Comment by Thermatix** [comment]
To be honest, I switched to using tagbar, and even then I don't tend to use it much, I find just dooing `/search_term` to be easier nowdays
